src/index.html

```diff
@@ -17,11 +17,381 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="style.scss"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
     />
```

Comment:
This link to the Google Fonts stylesheet is redundant and can be removed. You've already linked to the Google Fonts stylesheet at line 15.